src/thermo/Phase.cpp

```diff
@@ -25,6 +25,7 @@ Phase::Phase() :
     m_xml(new XML_Node("phase")),
     m_id("<phase>"),
     m_temp(0.001),
+    m_electron_temp(0.001),
```

Comment:
Make sure the order of this initialization corresponds to the order of declarations in `Phase.h` -- this gives a compiler warning "field 'm_electron_temp' will be initialized after field 'm_dens' [-Wreorder]".